server/src/main/java/com/linecorp/centraldogma/server/internal/storage/repository/git/GitRepository.java

```diff
@@ -129,30 +141,81 @@
      *
      * @param repoDir the location of this repository
      * @param repositoryWorker the {@link Executor} which will perform the blocking repository operations
+     * @param creationTimeMillis the creation time
      * @param author the user who initiated the creation of this repository
      *
      * @throws StorageException if failed to create a new repository
      */
-    GitRepository(Project parent, File repoDir, Executor repositoryWorker, Author author) {
+    GitRepository(Project parent, File repoDir, Executor repositoryWorker,
+                  long creationTimeMillis, Author author) {
+        this(parent, repoDir, GitRepositoryFormat.V1, repositoryWorker, creationTimeMillis, author);
+    }
+
+    /**
+     * Creates a new Git-backed repository.
+     *
+     * @param repoDir the location of this repository
+     * @param format the repository format
+     * @param repositoryWorker the {@link Executor} which will perform the blocking repository operations
+     * @param creationTimeMillis the creation time
+     * @param author the user who initiated the creation of this repository
+     *
+     * @throws StorageException if failed to create a new repository
+     */
+    GitRepository(Project parent, File repoDir, GitRepositoryFormat format, Executor repositoryWorker,
+                  long creationTimeMillis, Author author) {
```

Comment:
Could we add getter for the `creationTimeMillis` so that clients are able to get to know when this repository is created?

Reply:
Done